nightly.tests.cloudbuild.yaml

```diff
@@ -0,0 +1,46 @@
+steps:
+  - id: "Deploy Workflows"
+    name: "gcr.io/cloud-builders/gcloud:latest"
+    entrypoint: /bin/bash
+    allowFailure: true
+    args:
+      - "-c"
+      - |
+        for src in $(find ./src/ -type f -name *.workflows.yaml); do
+          tmp_name=${src:6}  # remove src/
+          tmp_name=${tmp_name/connectors\//}  # remove connectors/ if present
+          workflow_name=${tmp_name//./-}-$SHORT_SHA  # convert . to -
+
+          echo "Deploying $workflow_name with source: $src"
+
+          test="$(gcloud workflows deploy $workflow_name \
+              --source $src \
+              --location $_REGION \
+              --quiet 2>&1)"
+          
+          if [[ $? -ne 0 ]]; then
+              echo "$workflow_name (src: $src) failed!" >> /workspace/failures.txt
+              echo $test >> /workspace/failures.txt
```

Comment:
Could use [tee](https://notes.grayside.dev/posts/tools/gha-env-vars/) to directly output this here and skip the extra cat below. While the difference is small, it reduces the risk of losing the output if something strange happens and the "Check for Errors" step does not execute.